packages/expo-calendar/src/Calendar.ts

```diff
@@ -558,6 +558,135 @@ export type DaysOfTheWeek = {
 
 export { PermissionResponse, PermissionStatus, PermissionHookOptions };
 
+/**
+ * The result of presenting the calendar dialog for viewing or editing.
+ * These constants map to [`EKEventEditViewAction`](https://developer.apple.com/documentation/eventkitui/ekeventeditviewaction) and [`EKEventViewAction`](https://developer.apple.com/documentation/eventkitui/ekeventviewaction).
+ * @platform ios
+ * */
+export type CalendarDialogResultActionsIos = 'done' | 'canceled' | 'deleted' | 'responded';
```

Comment:
Hmm, is the description and values correct? 

On the referenced Apple docs page I do not see the `'canceled'` option:
* https://developer.apple.com/documentation/eventkitui/ekeventviewaction

It exist on `EKEventEditViewAction` tho:
* https://developer.apple.com/documentation/eventkitui/ekeventeditviewaction

Reply:
this type was replaced by an enum. This type indicates how user responded to the dialog for viewing or editing an event, which is why `saved` wasn't included. The new enum doesn't mention the corresponding types because it's not necessary.